userguide/tasks.rst

```diff
@@ -1793,23 +1791,37 @@ shows the configuration options for Cloud Syncs.
-   |                     |                     | for use as `salt <https://searchsecurity.techtarget.com/definition/salt>`__ for the encryption          |
-   |                     |                     | password. *Warning:* Save and back up the encryption salt value. Losing the salt value can result in    |
-   |                     |                     | data loss.                                                                                              |
+   | Filename encryption | checkbox            | Encrypt (*PUSH*) or decrypt (*PULL*) file names with the rclone `"Standard" file name encryption mode   |
+   |                     |                     | <https://rclone.org/crypt/#file-name-encryption-modes>`__. The original directory structure is          |
+   |                     |                     | preserved. Identical file names remain identical after encryption.                                      |
    |                     |                     |                                                                                                         |
+   |                     |                     | *PULL* tasks that have :guilabel:`Filename encryption` enabled and an incorrect                         |
+   |                     |                     | :guilabel:`Encryption password` or :guilabel:`Encryption salt` will not transfer any files but still    |
+   |                     |                     | report that the task was successful. To verify that files were transferred successfully, click the      |
+   |                     |                     | finished :ref:`task status <tasks_cloudsync_status_fig>` to see a list of transferred files.            |
+   +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
+   | Encryption password | string              | Password to encrypt and decrypt remote data. **Warning**: Always securely back up this password! Losing |
+   |                     |                     | the encryption password can result in data loss.                                                        |
+   +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
+   | Encryption salt     | string              | Enter a long string of random characters for use as                                                     |
+   |                     |                     | `salt <https://searchsecurity.techtarget.com/definition/salt>`__                                        |
+   |                     |                     | for the encryption password. **Warning**: Always securely back up the encryption salt value! Losing the |
+   |                     |                     | salt value can result in data loss.                                                                     |
```

Comment:
"Losing the salt value WILL result in data loss" is safer, don't give anybody false hope here ;)